src/trip/dto/create-trips-data.ts

```diff
@@ -0,0 +1,24 @@
+import { IsEnum, IsNotEmpty, IsUUID, MaxLength } from 'class-validator';
+
+export enum TripStatus {
+  NOT_STARTED = 'Not Started',
```

Comment:
```suggestion
  NOT_STARTED = 'not_started',
```
do same for other enums as well